irohad/torii/impl/command_service.cpp

```diff
@@ -207,7 +215,7 @@ namespace torii {
 
     log_->debug("StatusStream waiting finish, hash: {}", request_hash->hex());
 
-    if (not *finished) {
+    if (not*finished) {
```

Comment:
Formatting?

Reply:
It already was discussed upwards, formatting removes space between not and star

Reply:
That's very strange but okay